application/cmd/requester/main.go

```diff
@@ -0,0 +1,156 @@
+// Copyright 2020 Insolar Network Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"crypto"
+	"encoding/json"
+	"io/ioutil"
+	"net/url"
+	"os"
+
+	"github.com/insolar/insolar/application/api/requester"
+	"github.com/insolar/insolar/insolar/secrets"
+	"github.com/insolar/insolar/log"
+	"github.com/pkg/errors"
+	"github.com/spf13/pflag"
+)
+
+var (
+	memberKeysPath        string
+	apiURL                string
+	inputRequestParams    string
+	shouldPasteSeed       bool
+	shouldPastePrivateKey bool
+	memberPrivateKey      crypto.PrivateKey
+	request               *requester.ContractRequest
+)
+
+func parseInputParams() {
+	pflag.StringVarP(&memberKeysPath, "memberkeys", "k", "", "Path to member key paramsFile")
+	pflag.StringVarP(&apiURL, "url", "u", "", "API URL. for example http://localhost:19101/api/rpc")
+	pflag.StringVarP(&inputRequestParams, "request", "r", "", "The request body or path to request paramsFile")
+	pflag.BoolVarP(&shouldPastePrivateKey, "autocompleteseed", "s", false, "Should replace seed to correct value")
+	pflag.BoolVarP(&shouldPasteSeed, "autocompletekey", "p", false, "Should replace publicKey to correct value")
+	pflag.Parse()
+}
+
+func logFatal(args ...interface{}) {
+	log.Fatal(args)
```

Comment:
logFatal is not needed, better use panic instead
btw log.Fatal is Print() followed by os.Exit(1)